app/controllers/api/projects_controller.rb

```diff
@@ -0,0 +1,54 @@
+class Api::ProjectsController < ActionController::API
+
+  def show
+    @project = resource
+    @comments = @project.comments
+    @tasks = @project.tasks
+  end
+
+  def create
+    @workspace = parent
+    @project = @workspace.projects.build(project_params)
+    if  @project.save
+      render json: { status: 'SUCCESS', message: 'Project saved' }, status: :ok
+    else
+      render json: @project.errors, status: :unprocessable_entity
```

Comment:
return similar json to "success"

`json: { status: 'ERROR', errors: @project.errors}`